operator/api/v1beta1/lokistack_webhook.go

```diff
@@ -25,12 +28,17 @@ var _ webhook.Validator = &LokiStack{}
 
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (s *LokiStack) ValidateCreate() error {
-	return s.validate()
+	return s.validate(&LokiStack{})
```

Comment:
Why not `s.validate(s)` here?

Reply:
True, but what type of validations are applied at the end of the day against an empty object?

Reply:
Point. There are no comparisons to be done during the `create` step, so the empty object is passed. However, passing in the object to compare itself looks more odd than passing in an empty object.

Reply:
If there are no comparison-based validation, what is left to validate on the newly-created Lokistack object then?

Reply:
Hmm, nothing much. Alright, perhaps passing `nil` is the answer here. The `schemas` still need to be validated that there are unique dates and that there is a valid start date.